### PR TITLE
List networks sl

### PR DESCRIPTION
**Pull request by cdemergian**

### CODE REVIEW

**Comment by mihaitusa** [comment]
@cdemergian This commit is not ready for merge. First you have to fix all the problems that made this build fail.

**Comment by mihaitusa** [comment]
@cdemergian  Why in this branch (that is related to OpenStack) you have also date && time files (as new files)?

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - x509HostnameVerifier
+    - trustKeystore
+    - trustPassword
+    - keystore
+    - keystorePassword
+    - connectTimeout
+    - socketTimeout
+    - requestBody
+
+  action:
```
Comment (mihaitusa) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: replace here 'action' with 'java_action'

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    java_action:
```
Comment (mihaitusa) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: @cdemergian get rid of this line....

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      className: io.cloudslang.content.openstack.actions.ListNetworks
```
Comment (mihaitusa) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: @cdemergian  Replace here 'className' with 'class_name' and 'methodName' with 'method_name'

File: test/io/cloudslang/cloud/openstack/network/list_networks.inputs.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+trustAllRoots: "true"
+x509HostnameVerifier: ""
+trustKeystore: ""
+trustPassword: ""
+keystore: ""
+keystorePassword: ""
+connectTimeout: ""
+socketTimeout: ""
+requestBody: ""
+
```
Comment (mihaitusa) [inline test/io/cloudslang/cloud/openstack/network/list_networks.inputs.yaml]: add the local test suite as it is in test/cloud/heroku for e.g. and replace the hard-coded values with placeholders

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl (hunk 1 above)
Comment (cdemergian) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: done

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl (hunk 2 above)
Comment (cdemergian) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: done

File: content/io/cloudslang/cloud/openstack/network/list_networks.sl (hunk 3 above)
Comment (cdemergian) [inline content/io/cloudslang/cloud/openstack/network/list_networks.sl]: done

**Comment by mihaitusa** [comment]
This PR is now in oo-cs-rnd private repository

**Pull request closed without merging**